pkg/apis/networking/v1alpha1/service_types.go

```diff
@@ -30,6 +30,8 @@ const (
 	ResourcePluralMultiClusterService = "multiclusterservices"
 	// ResourceNamespaceScopedMultiClusterService indicates if MultiClusterService is NamespaceScoped.
 	ResourceNamespaceScopedMultiClusterService = true
+	// MCSServiceAppliedConditionType is indicates the condition type of mcs service applied.
+	MCSServiceAppliedConditionType = "ServiceApplied"
```

Comment:
Please move the events to `pkg/events/events.go`.

@jwcesign In my opinion, if we report a `warning` event and we need to report another `normal` event to cancel the event, right?

Reply:
ok

Reply:
I noticed that there is only a `Success` event and no `Failed` event.